utils/dockerutil.py

```diff
@@ -111,7 +122,11 @@ def get_events(self):
                 # [0]: https://github.com/docker/docker-py/pull/1082
                 log.debug('Unable to parse Docker event: %s', event)
 
-        return self.events, should_reload_conf
+        return self.events, conf_reload_set
+
+    def _get_checks_from_image(self, image):
+        """Get the list of checks applied to an image from the image_to_checks cache in the config store"""
+        return self.config_store.image_to_checks[image]
```

Comment:
Here we're relying on `image_to_checks` to be a `defaultdict` in case the image isn't in the cache. I think we should probably make that explicit either here or at the initialization of `self.image_to_checks` in `abstract_config_store`